Client/mods/deathmatch/logic/rpc/CVehicleRPCs.cpp

```diff
@@ -653,3 +654,25 @@ void CVehicleRPCs::SetVehiclePlateText(CClientEntity* pSourceEntity, NetBitStrea
         }
     }
 }
+
+
+void CVehicleRPCs::SetVehicleNitroActivated(CClientEntity* pSourceEntity, NetBitStreamInterface& bitStream)
+{
+    bool state;
+    if (!bitStream.ReadBit(state))
+        return;           
```

Comment:
I do not get it should not bitStream.ReadBit(state) check first if read was successful then return a value
and state is pass by reference

Reply:
In the case of ReadBit, this makes no sense, because such a condition will break the code if false. 

In the case of ReadBit, the result is the bit value, i.e. 0 (false) or 1 (true). Such a check makes sense only in the case of data such as Read, ReadVector, etc., but not in the case of ReadBit, because here, naturally, if you used WriteBit(false) earlier, the result of the ReadBit will also be false. There is no way for the read to fail here as it is only 1 bit